frontend/src/scenes/dashboard/dashboards/templates/dashboardTemplatesLogic.tsx

```diff
@@ -53,11 +67,19 @@ export const dashboardTemplatesLogic = kea<dashboardTemplatesLogicType>([
                 },
             },
         ],
+        allTemplates: [
+            [] as DashboardTemplateType[],
+            {
+                getAllTemplatesSuccess: (_, { allTemplates }) => allTemplates,
```

Comment:
This "reducer clause" should already be added through the loader. Adding your own reducer after just appends new things on top of the old one.